stdlib/src/strings.rs

```diff
@@ -113,6 +113,15 @@ pub unsafe extern "C" fn __quantum__rt__bigint_to_string(input: *const BigInt) -
     convert(&*input)
 }
 
+pub unsafe fn to_string(tag: *mut i8) -> String {
```

Comment:
I can't seem to find any consumers of this function... is it still needed or leftover from an earlier implementation?